AndrewTehan/hometask-3/cli.rb

```diff
@@ -0,0 +1,44 @@
+# frozen_string_literal: true
+
+ENTER_MESSAGE = 'Enter city name: '
+ERROR_MESSAGE = 'City name can not be blank!'
+EXIT_MESSAGE = 'exit!'
```

Comment:
I disagree with constants' scope. I'd move it to Cli class since it's used there.

A small note for the future: I'd recommend you avoid globalizing stuff since it's a bad practice.